config/release/dev/dev_patch.json

```diff
@@ -0,0 +1,6 @@
+[
+  {"op": "add",
```

Comment:
json patch is used instead of the strategic merge here to add the element to the array